<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JHEndoscope, Beijing Jinghai technology Co.Ltd</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="PageToolBar" name="pageToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="ImageSaveToolBar" name="imageSaveToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="VideoRecordToolBar" name="videoRecordToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="SettingsToolBar" name="settingsToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PageToolBar</class>
   <extends>QToolBar</extends>
   <header>PageToolBar.h</header>
  </customwidget>
  <customwidget>
   <class>ImageSaveToolBar</class>
   <extends>QToolBar</extends>
   <header>ImageSaveToolBar.h</header>
  </customwidget>
  <customwidget>
   <class>VideoRecordToolBar</class>
   <extends>QToolBar</extends>
   <header>VideoRecordToolBar.h</header>
  </customwidget>
  <customwidget>
   <class>SettingsToolBar</class>
   <extends>QToolBar</extends>
   <header>SettingsToolBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
